kafkaesk/consumer.py

```diff
@@ -229,8 +228,14 @@ async def finalize(self) -> None:
     async def _consumer_factory(self) -> aiokafka.AIOKafkaConsumer:
         consumer = self._app.consumer_factory(self.group_id)
         # This is needed in case we have a prefix
-        pattern = fnmatch.translate(self._app.topic_mng.get_topic_id(self.stream_id))
-        consumer.subscribe(pattern=pattern, listener=self)
+        topic_id = self._app.topic_mng.get_topic_id(self.stream_id)
+
+        if "*" in self.stream_id.split("."):
```

Comment:
So if we subscribe with something like:

```
foobartopic*
```

It will not translate into regex.

I don't think that is what we want right?

Should it just be: `if "*" in self.stream_id:`

Reply:
good idea.